ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -60,25 +58,21 @@ def _get_consumer(self) -> MessageBusConsumer:
 
     def process_message(self, message: str):
         """Callback method for MessageConsumer"""
-        Log.debug(f'Received the message from message bus: {message}')
-        result = False
-        # EventAnalyzer class instantiation is expected.
-        # This is not required. This will be called from EventAnalyzer class.
-        # But as its not available yet and to check newly added filter functionality,
-        # temporarily added this
-        result = self._cluster_resource_filter.filter_event(message.decode('utf-8'))
-        Log.info(f'Alert required: {result}')
+        Log.info(f'Received the message from message bus: {message}')
```

Comment:
should be log.debug  , otherwise it will flood the log

Reply:
Done